--- freetype-1.3.1/lib/arch/unix/Makefile.in.orig	2009-10-08 16:52:49.474105192 +0200
+++ freetype-1.3.1/lib/arch/unix/Makefile.in	2009-10-08 16:57:56.747409015 +0200
@@ -126,6 +126,7 @@
 # C flags
 CFLAGS = @CFLAGS@ @XX_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
+LDFLAGS = @LDFLAGS@
 FT_CFLAGS = $(CFLAGS) $(INCLUDES)
 
 # linker flags
@@ -195,7 +196,7 @@
                                         $(SRC_S)
 
 libttf.la: $(LIB_FILES)
-	$(LIBTOOL) --mode=link $(CC) -o libttf.la $(LIB_FILES) \
+	$(LIBTOOL) --mode=link $(CC) $(LDFLAGS) -o libttf.la $(LIB_FILES) \
                                -rpath $(libdir) \
                                -version-info $(version_info) $(FT_LIBS)
 
--- freetype-1.3.1/test/arch/unix/Makefile.in.orig	2009-10-08 16:52:49.472228742 +0200
+++ freetype-1.3.1/test/arch/unix/Makefile.in	2009-10-08 17:01:06.184056330 +0200
@@ -30,9 +30,10 @@
 
 CFLAGS = @CFLAGS@ @X_CFLAGS@ @XX_CFLAGS@
 CPPFLAGS = @CPPFLAGS@
+LDFLAGS = @LDFLAGS@
 FT_CFLAGS = $(INCDIRS) $(CFLAGS) $(CPPFLAGS) -DX11 -DLOCALEDIR='"@LOCALEDIR@"'
 
-FT_LIBS = @X_LIBS@ @X_PRE_LIBS@ @X_EXTRA_LIBS@ @LIBS@
+FT_LIBS = @X_LIBS@ @X_EXTRA_LIBS@ @LIBS@
 
 SRC = $(srcdir)/arabic.c           \
       $(srcdir)/common.c           \
@@ -62,7 +63,7 @@
 
 # variables used to compile either with libtool or not
 #
-PROCESS = $(LIBTOOL) --mode=link  $(CC) $(FT_CFLAGS)
+PROCESS = $(LIBTOOL) --mode=link  $(CC) $(LDFLAGS) $(FT_CFLAGS)
 #PROCESS = $(CC) $(FT_CFLAGS)
 
 LIBTTF = $(top_builddir)/lib/libttf.la
--- freetype-1.3.1/configure.in.orig	2009-10-08 16:52:50.204105629 +0200
+++ freetype-1.3.1/configure.in	2009-10-08 17:00:45.140723630 +0200
@@ -41,6 +41,7 @@ else
   esac
 fi
 AC_SUBST(XX_CFLAGS)
+AC_SUBST(LDFLAGS)
 
 dnl at least Digital UNIX 4.0d needs this due to a strange make program
 
